_data/nikunj3011/nikunj3011.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Nikunj Rathod
+institution: New Brunswick Community College
+profile_pic: nikunj.jpg
+quote: "Just another human being who wants to know how universe works with computers, games and algorithms."

```

Comment:
Hey @nikunj3011 , can you avoid using ( " ) in quote just to keep the format intact ?

Reply:
ok